rdmo/projects/constants.py

```diff
@@ -10,3 +10,10 @@
     (ROLE_AUTHOR, _('Author')),
     (ROLE_GUEST, _('Guest')),
 )
+
+VISIBILITY_PRIVATE = 'private'
```

Comment:
you don't wanna try something like this for the constants?

```py
class VisibilityEnum(Enum):
    PRIVATE = 'private'
    INTERNAL = 'internal'

    @classmethod
    def choices(cls):
        return [(tag.value, _(tag.name.capitalize())) for tag in cls]
```

Reply:
Django has actually also the Enum pattern built in, https://docs.djangoproject.com/en/4.2/ref/models/fields/#enumeration-types., and
>  Changed in Django 5.0:
> Support for mappings and callables was added.

ok better to keep it like that then, think it's the same principle as
> Explicit is better than implicit.

Reply:
The `models.TextChoices` is interesting, but I want to keep it uniformly in RDMO. We might change that in the future for all choice fields.